MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -120,6 +120,23 @@ void UuidHandler(AtomCallbackArgs a)
                 }
             }
 
+            void UserDataHandler(AtomCallbackArgs a)
+            {
+                switch (a.TypeString)
+                {
+                    case "?xyz":
+                        var directory = new QuickTimeMetadataHeaderDirectory();
+                        var stringSize = a.Reader.GetUInt16();
+                        var languageCode = a.Reader.GetUInt16();
```

Comment:
Is this code interesting? If not, maybe `Skip(2)` instead (with a comment "skip language code" or similar).

Reply:
Nope, those values aren't useful (at least IMO) and could be skipped past.